server/utils/blackjack_deck.py

```diff
@@ -73,8 +73,6 @@ def blackjack_total(hand):
     return total
 
 def get_deck_set():
-    if RANDOM_ORG_KEY == "DEFAULT_KEY":
```

Comment:
How are you handling the absence of random org key now?

Reply:
if I put an invalid key in the https request it returns json with "error" key. Line 92 checks for json with "error" in it for a errors on the website like reached number of calls, etc. I found out after testing that bad keys count under that if statement.